requirements.txt

```diff
@@ -0,0 +1,4 @@
+aiohttp==1.0.5
+pytest==3.0.4
+requests==2.12.1
+responses==0.5.1
```

Comment:
Follow-up: For your *tests*, to make them reproducible and explicit you may want to pin everything quite rigorously to known versions.